api/python/provisioner/salt.py

```diff
@@ -594,7 +595,7 @@ def run(
             try:
                 logger.debug(
                     "Function '{}' on '{}' resulted in {}"
-                    .format(fun, targets, res.results)
+                    .format(fun, targets, pformat(res.results))
```

Comment:
Can we use f-string for consistency?

Reply:
Only in places where pretty format `pformat` is used, I've changed to `{}.format()'. Other places have to f-string.

Reply:
Does this not work?

```
logger.debug(
    f"Function '{fun}' on '{targets}' resulted in {pformat(res.results)}"
)
```
Try if it helps.

Reply:
It worked, but since it was a type of `format()`, just kept it for similarity. 
Updated now, also in a few more places. Thanks !